src/services/keymanager.ts

```diff
@@ -121,6 +122,15 @@ interface KeyManager {
         password: string,
         salt?: Checksum256
     ): Promise<{ privateKey: PrivateKey; salt: Checksum256 }>;
+
+    /**
+     * checks the key against the provided challenge with hashedSaltedChallenge
+     * @param Options for checking key with level, and challenge
```

Comment:
show type and variable name like:
@param options {GetKeyOption} - 

also add a typedef to GetKeyOption

https://devhints.io/jsdoc